README.md

```diff
@@ -671,8 +710,9 @@ The `robotology-superbuild` is based on [YCM](https://github.com/robotology/ycm)
 Mantainers
 ==========
 
-| Profile  | Mantainer                     |
+| Profile  | Maintainer                     |
 |:--------:|:-----------------------------:|
 | Core     | Silvio Traversaro [@traversaro](https://github.com/traversaro) |
 | Dynamics | Silvio Traversaro [@traversaro](https://github.com/traversaro) |
 | Teleoperation | Kourosh Darvish [@kouroshD](https://github.com/kouroshD) |
+| Human Dynamics | Yeshasvi Tirupachuri [@Yeshasvitvs](https://github.com/Yeshasvitvs) |
```

Comment:
@Yeshasvitvs are you ok with this? : )

Reply:
I remember I asked @lrapetti  and @Yeshasvitvs  before, and it was fine for them.

Reply:
Yes